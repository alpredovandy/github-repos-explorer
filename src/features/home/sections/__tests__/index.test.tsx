// import { render } from '@testing-library/react';
// import { describe, expect, it, vi } from 'vitest';

// import { MOCK_GIFS } from '@/domains/__mocks__/Gif.ts';
// import type { CatalogProps } from '@/features/home/sections/types';

// import Catalog from '../Catalog';

// describe('Test Home/section: Catalog', () => {
//     // @ts-ignore
//     const renderCatalog = (props?: CatalogProps) => render(<Catalog gifs={MOCK_GIFS.data ?? []} {...props} />);

//     it('Snapshot', () => {
//         const { container } = renderCatalog();

//         expect(container).toMatchSnapshot();
//     });

//     it('should show loading when has isPending props true', () => {
//         renderCatalog({
//             gifs: [],
//             isPending: true,
//             loadButton: vi.fn(),
//             showButton: true,
//         });
//     });
// });
