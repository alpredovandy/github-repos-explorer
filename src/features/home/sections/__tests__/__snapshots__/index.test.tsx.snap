// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Test Home/section: Catalog > Snapshot 1`] = `
<div>
  <div
    class="grid flex-auto md:grid-cols-2 xl:grid-cols-3 flex-wrap gap-2"
  >
    <div
      class="card w-46 bg-white shadow-xl"
    >
      <img
        alt="Shoes"
        class="object-fill h-40 w-full rounded relative"
        src="https://media2.giphy.com/media/3o6wrvdHFbwBrUFenu/giphy-preview.gif"
      />
      <button
        class="absolute top-0 right-0 left-0 bottom-0 text-white font-bold"
      >
        pause
      </button>
    </div>
  </div>
  <button
    class="btn btn-active btn-primary items-center text-lg"
  >
     
    Load more
  </button>
</div>
`;
